src/test/java/spoon/test/comment/CommentTest.java

```diff
@@ -263,7 +264,12 @@ public void testInLineComment() {
 		assertEquals(createFakeComment(f, "comment after array value"), arrayValue.getComments().get(1));
 
 
-		CtReturn ctReturn = m1.getBody().getStatement(12);
+		CtLocalVariable ctLocalVariableString = m1.getBody().getStatement(12);
+		assertEquals(createFakeComment(f, "comment multi line string"), ((CtBinaryOperator)((CtBinaryOperator)ctLocalVariableString.getDefaultExpression()).getRightHandOperand()).getLeftHandOperand().getComments().get(0));
+		assertEquals("\"\" + (\"\"// comment multi line string" + newLine
+				+ " + \"\")", ctLocalVariableString.getDefaultExpression().toString());
+
+		CtReturn ctReturn = m1.getBody().getStatement(13);
 		assertEquals(createFakeComment(f, "comment return"), ctReturn.getComments().get(0));
```

Comment:
It is complex...
Now I print comments of CtExpression.

These lines was basically there to handle CtExpressions present in CtConditional (may not work for other expression).
```java
if (!(thenExpression instanceof CtStatement)) {		
 	elementPrinterHelper.writeComment(thenExpression, CommentOffset.BEFORE);		
}
```
Now I handle the comment of every CtExpression.

More over I added a position for the string concatenation.
This way JDTCommentBuilder is able to handle the comment in string.